composer.json

```diff
@@ -27,7 +27,8 @@
     "prefer-stable": true,
     "minimum-stability": "dev",
     "require": {
-        "php": "^8.0, <8.3"
+        "php": "^8.0, <8.3",
+        "gacela-project/dependency-resolver": "^0.1"
```

Comment:
Thinking about the package name... 🤔 

Some proposals:
- `Dependency Resolver`
- `Dependency Container`
- `Dependency Injector`
- `Service Container` 
- `Container` -> [Laravel](https://laravel.com/docs/10.x/container)
- `Dependency Injection` -> [Symfony](https://symfony.com/doc/current/components/dependency_injection.html)
- `Object Resolver` (was the name proposed by @Chemaclass?)
- you say ...!

I think I like `Dependency Resolver`, but open to discuss.
But I think we should add these proposals to the `keywords` key on L13 to have more visibility, what do you think?

Reply:
Resolver ☝🏻


Another option would be Binder.

Reply:
I like that. I renamed it to just "resolver" -> `gacela-project/resolver`